### ISSUE TITLE
Reactions: "content" parameter is an enum

### ISSUE THREAD

**Issue by gr2m** [octokit/routes#96]
- endpoint scope: Reactions
- endpoint name: Pretty much all of them, but for example "List reactions for an issue" and "Create reaction for an issue"
- documentation URL: https://developer.github.com/v3/reactions

### Current specifications

https://github.com/octokit/routes/blob/a509e1a89672025ec580405648b9c5666e3e7b06/routes/reactions/list-reactions-for-an-issue.json#L25-L30

https://github.com/octokit/routes/blob/a509e1a89672025ec580405648b9c5666e3e7b06/routes/reactions/create-reaction-for-an-issue.json#L25-L30

### I expected

The `"content"` parameter should be an an enum with the options 

```json
[
  "+1",
  "-1",
  "laugh",
  "confused",
  "heart",
  "hooray"
]
```

If you’d like to work on it, just comment below and follow instructions in [CONTRIBUTING.md](https://github.com/octokit/routes/blob/master/CONTRIBUTING.md)

**Comment by knrt10** [comment]
@gr2m I would like to work on this issue. And just for confirmation 

**Expected outcome should be**

```
{
  "name": "List reactions for an issue",
  "enabledForApps": true,
  "method": "GET",
  "path": "/repos/:owner/:repo/issues/:number/reactions",
  "params": [
    {
      "name": "owner",
      "type": "string",
      "required": true,
      "description": ""
    },
    {
      "name": "repo",
      "type": "string",
      "required": true,
      "description": ""
    },
    {
      "name": "number",
      "type": "integer",
      "required": true,
      "description": ""
    },
    {
      "name": "content",
      "type": "string",
      "description": "Returns a single [reaction type](/v3/reactions/#reaction-types). Omit this parameter to list all reactions to an issue.",
      "required": false,
	  "enum": [
        "+1",
	    "-1",
	    "laugh",
	    "confused",
	    "heart",
	    "hooray"
      ]
    },
    {
      "name": "per_page",
      "type": "integer",
      "required": false,
      "description": "Results per page (max 100)",
      "default": 30
    },
    {
      "name": "page",
      "type": "integer",
      "required": false,
      "description": "Page number of the results to fetch.",
      "default": 1
    }
  ],
  "description": "List the reactions to an [issue](/v3/issues/).",
  "documentationUrl": "https://developer.github.com/v3/reactions/#list-reactions-for-an-issue"
}

```

This is for ```routes/routes/reactions/create-reaction-for-an-issue.json```

Am I right?

**Comment by gr2m** [comment]
looks good 👍

**Comment by knrt10** [comment]
@gr2m you can now close #94 #95 this #98 and #99.

**Comment by gr2m** [comment]
fixed via #105
